tools/ansible/tasks/wsu/main.yaml

```diff
@@ -2,29 +2,131 @@
 - hosts: localhost
   vars:
     project_root: "{{ playbook_dir }}/../../../.."
-    kubelet_location: "https://dl.k8s.io/v1.16.2/kubernetes-node-windows-amd64.tar.gz"
     wmcb_exe: "wmcb.exe"
+    hybrid_overlay_exe : "hybrid-overlay.exe"
+    build: False
+    # Dictionary mapping Kubernetes version to the corresponding OpenShift Version
+    map_kubernetes_to_openshift_version:
+      '1.16': 4.3
+      '1.17': 4.4
+      '1.18': 4.5
```

Comment:
Why did you remove the `Get cluster version` task and replace it with this? Doing this makes the playbook unusable after kubernetes 1.18. 
While I dont see it being used at that point, I'd rather not have a timebomb built into our code.

Reply:
Your point is valid. My thinking when deciding to move to map was to consider the scenario when OpenShift version skips a kubernetes version, like the way it happened in 4.3, and it would be easier to maintain a map rather than changing the script

Reply:
@suhanime I am :-1: on this change, if you think this is a better way of doing things, we can get a third opinion, else please revert this change :)

Reply:
I think it is easier to manage versions we support via a map. @ravisantoshgudimetla @aravindhp @akhil-rane please weigh in to help us decide. The discussion is about dropping (Get cluster version task)[https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/tools/ansible/tasks/wsu/main.yaml#L88-L104] and using a map of kubernetes version to cluster version instead

Reply:
I am not a fan of either method mainly because we are using them so that things would work in CI. Given we are back here, I suggest to add a proper way that works for the "product" and we have a fall back hack that works for CI. That fall back hack should be simple and easy to maintain.

Reply:
We can use `oc version` here to get openshift version as well, and use the hack if the output is not in the format we expect (like 4.x) - does that seem good enough for the product?

Reply:
Using the `oc version` is fine with me. Others can chime in here too.

Reply:
To be clear, we shouldn't add something to product for it to work in CI. More code means more bugs. Given that, I am fine with having a map which gets updated for every release. Ideally, we would cut a new release in our repo for every release of OpenShift so that there is one-to-one mapping. We don't have to maintain this map going forward.